### PR TITLE
Resolved issue #4

### PR DESCRIPTION
**Pull request by camaechi**
CTRL + W to run faster

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -2413,12 +2413,17 @@ void handle_movement(double dt) {
...
     int sx = 0;
+    int isRunning = 0;
     if (!g->typing) {
         float m = dt * 1.0;
         g->ortho = glfwGetKey(g->window, CRAFT_KEY_ORTHO) ? 64 : 0;
         g->fov = glfwGetKey(g->window, CRAFT_KEY_ZOOM) ? 15 : 65;
         if (glfwGetKey(g->window, CRAFT_KEY_FORWARD)) sz--;
         if (glfwGetKey(g->window, CRAFT_KEY_BACKWARD)) sz++;
+        if (glfwGetKey(g->window, CRAFT_KEY_FORWARD)) {
+            if(glfwGetKey(g->window, CRAFT_KEY_RUN)) isRunning = 1;
```
Comment (TonePoems) [inline src/main.c:2424]: I would suggest including this under the existing `CRAFT_KEY_FORWARD` if statement as it is duplicated code

File: src/main.c [hunk 2]
```diff
@@ -2439,6 +2444,7 @@ void handle_movement(double dt) {
         }
     }
     float speed = g->flying ? 20 : 5;
+    if(isRunning) speed*=4;
```
Comment (TonePoems) [inline src/main.c:2447]: I thought a multiplier of 1.5 felt better when I tested it. Reminded me of minecrafts sprint; *4 seems like with speed boosts. But of course that is just preference

**Review by TonePoems — CHANGES_REQUESTED** [review]
Will also need to have tests eventually

**Comment by TonePoems** [comment]
Oh this should also be added to the readme section on controls

**Comment by dburd08** [comment]
capturing comments from team discord:
need tests of some kind, we have another story this sprint for setting up a test suite.
Do we want to apply the speed multiplier in all directions of movement or only in the 'forward' direction

**Pull request merged**